docker/WORKSPACE.cu111.docker

```diff
@@ -0,0 +1,94 @@
+workspace(name = "TRTorch")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "rules_python",
+    remote = "https://github.com/bazelbuild/rules_python.git",
+    commit = "4fcc24fd8a850bdab2ef2e078b1de337eea751a6",
+    shallow_since = "1589292086 -0400"
+)
+
+load("@rules_python//python:repositories.bzl", "py_repositories")
+py_repositories()
+
+load("@rules_python//python:pip.bzl", "pip_repositories", "pip3_import")
+pip_repositories()
+
+http_archive(
+    name = "rules_pkg",
+    url = "https://github.com/bazelbuild/rules_pkg/releases/download/0.2.4/rules_pkg-0.2.4.tar.gz",
+    sha256 = "4ba8f4ab0ff85f2484287ab06c0d871dcb31cc54d439457d28fd4ae14b18450a",
+)
+
+load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
+rules_pkg_dependencies()
+
+git_repository(
+    name = "googletest",
+    remote = "https://github.com/google/googletest",
+    commit = "703bd9caab50b139428cea1aaff9974ebee5742e",
+    shallow_since = "1570114335 -0400"
+)
+
+# CUDA should be installed on the system locally
+new_local_repository(
+    name = "cuda",
+    path = "/usr/local/cuda-11.1/",
```

Comment:
I think we can use a single WORKSPACE file for all dockerfiles if we make this `/usr/local/cuda`

Reply:
Done, nice suggestion!